pyat/at/matching/globalfit.py

```diff
@@ -73,7 +76,8 @@ def _fit(ring, index, func, refpts1, refpts2, newval, J,
 
 def fit_tune(ring: Lattice, refpts1: Refpts, refpts2: Refpts, newval,
              tol: float = 1.0e-12,
-             dp: Optional[float] = 0, niter: int = 3, regex=False,
+             dp: Optional[float] = 0, niter: int = 3, regex=False, 
+             delta: Optional[float] = None,
```

Comment:
Here, I would call it `KStep`, by analogy with `XYStep` and `DPStep` already defined, and call `_fit_tunechrom(..., delta=KStep,...)`

If you want to add these constants in `at.DConstant`, it will be easier to remove the explicit argument and use `kwargs`:
```python
delta = kwargs.pop('KStep', DConstant.KStep)    # if defined in at.DConstant
...
_fit_tunechrom(..., delta=delta)
```